lq1/default.cfg

```diff
@@ -111,4 +107,58 @@ alias +lq-zoom90 "fov 90;wait;fov 80;wait;fov 70;wait;fov 60;wait;fov 50;wait;fo
 alias -lq-zoom90 "fov 50;wait;fov 60;wait;fov 70;wait;fov 80;wait;fov 90"
 alias +lq-zoom120 "fov 110;wait;fov 100;wait;fov 90;wait;fov 80;wait;fov 70;wait;fov 60;wait;fov 50;wait;fov 40"
 alias -lq-zoom120 "fov 50;wait;fov 60;wait;fov 70;wait;fov 80;wait;fov 90;wait;fov 100;wait;fov 110;wait;fov 120"
+
+// Graphics Style Toggle
+alias lq_style_modern_exec "exec style_modern.cfg"
+alias lq_style_retro_exec "exec style_retro.cfg"
+alias lq_style_modern "lq_style_modern_exec; alias lq_toggle_style_set lq_style_retro"
+alias lq_style_retro "lq_style_modern_exec; alias lq_toggle_style_set lq_style_modern"
+alias lq_toggle_style "cl_warncmd 0; lq_toggle_style_set; exec autoexec.cfg; vid_reload; cl_warncmd 1"
+lq_style_modern
+
+// Other Config Files
+alias	"cam"		"exec cam.cfg"
+alias	"dev"		"exec dev.cfg"
+alias	"zoom90"	"exec zoom90.cfg"
+alias	"zoom120"	"exec zoom120.cfg"
+
+
+// ================================
+// Compatibility With Other Engines
+// ================================
+
+// Ironwail

```

Comment:
Shouldn't this be in an ironwail specific config file?

Reply:
Does Ironwail have its own config file other than ironwail.cfg? ironwail.cfg isn't meant to be distributed since it gets automatically written by the engine to save your settings. Also, a lot of these commands work in other QuakeSpasm forks.

Reply:
`ironwail.cfg` is just a sub for `config.cfg`. Yes, it is overwritten when you apply new settings, it is my understanding though that it should preserve aliases?

It's definitely worth investigating organizing these better. I know it's not a problem you've created but it is a mess. It may not "break" anything but we can't actually be sure and we know it will cause really issues on many engines where it's forced to go through all of these unregistered cvars and spamming the console. 

If we had a better backbone for this it would make supporting enhanced features of other engines significantly more streamlined both if we ever want to create more configs/aliases ourselves and if outside contributors want to add that support.

Reply:
> Shouldn't this be in an ironwail specific config file?

we don't ship engine specific configs so no